package.json

```diff
@@ -50,7 +50,8 @@
   },
   "scripts": {
     "install": "prebuild --download",
-    "test": "tape test/*-test.js | faucet"
+    "test": "tape test/*-test.js | faucet",
+    "prebuild": "rm -rf prebuilds && prebuild --verbose"
```

Comment:
instead of always removing the prebuilds folder the user could just pass `--force` to force a rebuild